Add some traits for cocountable topology on $\mathbb{R}$ (#1132)
pi-base/data: spaces/S000017/README.md

  - wikipedia: Cocountable_topology
    name: Cocountable topology on Wikipedia
---

> Let $X=\mathbb R$ be the set of real numbers.  Define topology on $X$ by letting a subset $U \subset X$ be open iff its complement is countable or $U = \emptyset$. 

Reviewer: grammar: "the topology"

> Let $X=\mathbb R$ be the set of real numbers.  Define the topology on $X$ by letting a subset $U \subset X$ be open iff its complement is countable or $U = \emptyset$. 

Reviewer: ```suggestion
Let $X=\mathbb R$ be the set of real numbers.  Define the topology on $X$ by letting a set in $X$ be open iff it is empty or its complement is countable. 
```
further streamlining
